src/modules/browser/speech-synthesis/useBrowserSpeechVoiceDropdown.tsx

```diff
@@ -0,0 +1,124 @@
+import * as React from 'react';
+
+import { CircularProgress, Option, Select } from '@mui/joy';
+import KeyboardArrowDownIcon from '@mui/icons-material/KeyboardArrowDown';
+import RecordVoiceOverTwoToneIcon from '@mui/icons-material/RecordVoiceOverTwoTone';
+
+import { useBrowseVoiceId } from './store-module-browser';
+import { speakText, cancel } from './browser.speechSynthesis.client';
+
+function VoicesDropdown(props: {
```

Comment:
I see this is a duplication of the ElevenLabs. probalby needed because of different logic.